server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/AbstractRecipientRewriteTableMailet.java

```diff
@@ -19,22 +19,28 @@
 

```

Comment:
In fact, it still has more sub class of that abstract class: UsersRepositoryAliasingForwarding even it is deprecated but as discussed at the beginning, we keep it. That's why I keep the abstract class. For further, if we decide to delete UsersRepositoryAliasingForwarding, so this abstract class won't exist.